### PR TITLE
resolved golint errors for variable names, stale comments and more

### PR DESCRIPTION
**Pull request by girishramnani**
## What is the purpose of this change? What does it change?

Resolved golint errors related to:

- Variable name formating ( Url -> URL )
- Comments incorrectly written ( function name not matching the comment heads )
- redundant `else` for if statements having return at the `end`

## Was the change discussed in an issue?
Not discussed

## How to test changes?

changes do not impact functionality

### CODE REVIEW

File: tests/e2e/examples/source/dotnet/Startup.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public static void Main(string[] args)
                     options.NoDelay = true;
                     options.UseConnectionLogging();
                 })
-                .UseUrls("http://0.0.0.0:8080")
+                .UseURLs("http://0.0.0.0:8080")
```
Comment (geoand) [inline tests/e2e/examples/source/dotnet/Startup.cs]: This is not correct, it breaks the source e2e test :)

Comment (girishramnani) [reply tests/e2e/examples/source/dotnet/Startup.cs]: ack

**Comment by geoand** [comment]
@girishramnani Looks good! Since you're in the code quality improvement mode :), could you please also address the 2 new issues CodeClimate is reporting?

**Comment by girishramnani** [comment]
The code climate issue is related to not having code comments, I can open a
new PR specifically addressing all absent code comments issue or I can
solve this here itself? Open to suggestions.

**Comment by geoand** [comment]
I would suggest that this PR just fix the 2 new issues that are reported - obviously it doesn't make sense to try and address all of them :)

**Comment by girishramnani** [comment]
resolved @geoand

**Pull request merged**